test/fixtures/templater/jinja_f/jinja.yml

```diff
@@ -111,7 +111,7 @@ file:
                   end_square_bracket: "]"
               keyword: AS
               data_type:
-                data_type_identifier: FLOAT64
```

Comment:
If you look at the `.sqlfluff` file for this test it's using ANSI dialect so we shouldn't change this for jinja_f yet. Then the tests will pass. (you were right to change it for jinja_g_macros though as that uses postgres 👍 )